packages/vscode/package.json

```diff
@@ -9,7 +9,7 @@
     "theme": "dark"
   },
   "version": "0.8.8",
-  "author": "Astro",
```

Comment:
I changed it here too but I'm not sure if it can cause issues with publishing? I checked the VS Code documentation regarding this and it said it wouldn't cause any issues though

Reply:
Shouldn't cause any problems! Code uses the `publisher` key, which is "astro-build".